.ci/build.sh

```diff
@@ -80,16 +80,17 @@ function build_it {
   # ensure pyenv shims are added to PATH, see https://github.com/pyenv/pyenv/issues/1906
   eval "$(pyenv init --path)"
   eval "$(pyenv virtualenv-init -)"
-  pip install opensearch-benchmark
-  docker pull datadog/squid:latest
+  python3 --version
```

Comment:
Is this just for verbose output?  Perhaps adding an `echo "Python version is .."` might be more helpful.